frontend/packages/knative-plugin/src/utils/__tests__/resource-overview-utils.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { MockKnativeResources } from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
+import { getKnativeRoutesLinks } from '../resource-overview-utils';
+
+describe('resource overview utils', () => {
+  const multipleRevisionsData = {
+    ...MockKnativeResources.ksroutes.data[0],
+    status: {
+      ...MockKnativeResources.ksroutes.data[0].status,
+      traffic: [
+        {
+          ...MockKnativeResources.ksroutes.data[0].status.traffic[0],
+          percent: 50,
+          tag: 'tag-one',
+          url: 'http://testing.apps.bpetersen-june-23.devcluster.openshift.com',
+        },
+        {
+          ...MockKnativeResources.ksroutes.data[0].status.traffic[0],
+          percent: 25,
+        },
+        {
+          ...MockKnativeResources.ksroutes.data[0].status.traffic[0],
+          revisionName: 'test',
+          percent: 25,
+        },
+      ],
+    },
+  };
+  it('expect getKnativeRoutesLinks to return array of resource overview item', () => {
+    const routeLinks = getKnativeRoutesLinks(
+      MockKnativeResources.ksroutes.data[0],
+      MockKnativeResources.revisions.data[0],
+    );
+    expect(routeLinks).toBeDefined();
```

Comment:
`expect(routeLinks).toBeDefined();` this will always be defined right? so can we avoid this